.github/workflows/deploy_to_s3_prod.yml

```diff
@@ -0,0 +1,41 @@
+# This workflow will build the vuepress static files
+# and upload them to the given s3 bucket
+name: Deploy to prod
+
+on:
+  push:
+    branches: ["release"]
+  
+jobs:
+  build_and_deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Cache node modules
+      uses: actions/cache@v1
+      with:
+        path: node_modules
+        key: ${{ runner.OS }}-build-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.OS }}-build-
+          ${{ runner.OS }}-
```

Comment:
why do we need this?

Reply:
https://docs.github.com/en/actions/guides/caching-dependencies-to-speed-up-workflows